index.bs

```diff
@@ -1395,10 +1404,6 @@ Build a composed graph up to a given output operand into a computational graph a
         1. If |operand| is in [=this=]'s [=MLGraphBuilder/graph=]'s [=computational graph/inputs=], [=set/append=] |operand| to |inputs|.
         1. [=list/For each=] |input| of |operand|.{{MLOperand/[[operator]]}}'s [=operator/inputs=]:
             1. [=queue/Enqueue=] |input| to |queue|.
```

Comment:
Note: this PR does not address https://github.com/webmachinelearning/webnn/issues/567#issuecomment-1996062048 since there didn't appear to be consensus on whether that was desirable (FYI @inexorabletash @fdwr). If folks have strong feelings in either direction then I propose we file a separate issue to discuss; otherwise I propose we leave this as-is for now. WDYT?

Reply:
SGTM.

Reply:
Even with this change, we can still AFAICS create a disconnected graph with a single graphBuilder, right? It's a legitimate graph construct.

![image](https://github.com/webmachinelearning/webnn/assets/1809166/5bb8fa33-d4bd-463b-97c5-261b7b57511b)